templates/deployment-worker.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: airflow
+        tier: worker
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: worker
+        image: eanylin/airflow-helm:1.8.1-1.6.2
```

Comment:
Same.  This needs to be a public image produced by github.com/att-comdev/dockerfiles/airflow